cli/src/commands.rs

```
@@ -23,10 +23,30 @@ pub struct CommandContext {
     pub client: Client,
 }
 
+/// Error returned by [CommandT::run].
+///
+/// Wraps the message that is displayed to the user. We provide [From] instances for client [Error]
+/// and [DispatchError].
+#[derive(Clone, Debug)]
+pub struct CommandError(pub String);
```

I am aware this is the CI and that these errors are not propagated down the road. However, we would benefit from having `CommandError` as an enum, where the variants would be: 

```enum CommandError {
    ClientError(Error).
    FailedTransaction(Block),
    InvalidProjectName(Error),
    ProjectNotFound
```

By then implementing `Display`, we would have a single place defining how these errors are displayed back to the user. Now we do it every time we build `CommandError`, a duplication that easily leads to a bad ux shall we happen to change one message and not keep the others consistent. Besides that point, it's also beneficial to have this type informing of all error types a user can encounter when interaction with our cli.

---

Yes, good piont. I’m using this approach now.